### PR TITLE
WIP: remote-access/vnc - improve page

### PR DESCRIPTION
**Pull request by andrum99**
Quite often on the forums, users will have trouble enabling VNC when there is no display attached to the Pi. Address this by improving the VNC documentation, and take the opportunity to make some other minor improvements as well.

### CODE REVIEW

**Comment by andrum99** [comment]
Ready for review.

File: remote-access/vnc/README.md [hunk 1]
```diff
@@ -1,86 +1,104 @@
...
-### Establishing a direct connection
+In this mode, VNC Server will create a graphical desktop in the memory of the Raspberry Pi: this allows you to use the graphical desktop of the Pi without having its display enabled.
 
-Direct connections are quick and simple providing you're joined to the same private local network as your Raspberry Pi. For example, this might be a wired or wireless network at home, at school, or in the office.
+![Connecting to an in-memory virtual desktop](images/raspberry-pi-virtual.png)
+
+A virtual desktop can be created on the Raspberry Pi as follows:
+
+- At the command line, run `vncserver`
+- Make a note of the IP address and display number shown, for example `192.168.5.25:1` indicates IP address 192.168.1.215 and display number 1
```
Comment (JamesH65) [inline remote-access/vnc/README.md:64]: Different IP addresses?

File: remote-access/vnc/README.md [hunk 2]
```diff
@@ -1,86 +1,104 @@
...
-
-You must enable VNC Server before you can use it: instructions for this are given below. By default, VNC Server gives you remote access to the graphical desktop that is running on your Raspberry Pi, as though you were sitting in front of it.
+VNC Connect from [RealVNC](realvnc.com) is included with Raspberry Pi OS. It consists of both VNC Server, which allows you to control your Raspberry Pi remotely, and VNC Viewer, which allows you to control other computers remotely from your Raspberry Pi.
 
-However, you can also use VNC Server to gain graphical remote access to your Raspberry Pi if it is headless or not running a graphical desktop. For more information on this, see **Creating a virtual desktop**, further below. 
+VNC Server can operate in two different ways on the Raspberry Pi. The standard method, called *service mode*, allows remote access to a display which is enabled on the Raspberry Pi itself: you will be unable to connect if there is no display enabled. A second method is to instead use a *virtual desktop*: a graphical desktop is created in memory, and remote devices connect to this.
```
Comment (JamesH65) [inline remote-access/vnc/README.md:10]: "in memory on the Raspberry Pi" ??

File: remote-access/vnc/README.md [hunk 3]
```diff
@@ -1,86 +1,104 @@
...
-- Scroll down and select **VNC > Yes**.
+The VNC system service is named `vncserver-x11-serviced`.
 
-## Connecting to your Raspberry Pi with VNC Viewer
 
-There are two ways to connect to your Raspberry Pi. You can use either or both, depending on what works best for you.
+## Virtual desktop mode
 
-### Establishing a direct connection
+In this mode, VNC Server will create a graphical desktop in the memory of the Raspberry Pi: this allows you to use the graphical desktop of the Pi without having its display enabled.
```
Comment (JamesH65) [inline remote-access/vnc/README.md:57]: Do we need some description of what happens if the remote Pi does not have a connected display and you try to connect with VNC? So people have more indication they need to use virtual mode?

Comment (andrum99) [reply remote-access/vnc/README.md:57]: I've added a description of what happens in this case - not sure if a more prominent warning is needed - this seems to trip up quite a few folk on the forums.

Comment (andrum99) [reply remote-access/vnc/README.md:57]: Also, please don't integrate this one yet - I'm going look at the proposed changes in #1628, and I think an image of the VNC desktop icon would be useful so folk know what to look for. I will take the WIP tag off once I've done this.

**Pull request closed without merging**